docs/CONTRIBUTING.md

```diff
@@ -50,6 +50,18 @@ See [reporting of security vulnerabilities](https://systemd.io/SECURITY).
 * Github will automatically add the `please-review` label when a pull request is opened or updated.
   If you need more information after a review, you can comment `/please-review` on the pull request to have Github add the `please-review` label to the pull request.
 
+## Using AI Code Generators
```

Comment:
As far as I understand it was prompted by PRs opened recently but I wonder if by analogy with curl it would make sense to also cover issues and things like that: https://github.com/curl/curl/blob/master/docs/CONTRIBUTE.md#on-ai-use-in-curl? I point to that policy in cases like https://github.com/google/oss-fuzz/issues/13142 and it seems it helps when there are actual people behind issues/PRs/....